service/sandbox-core/src/main/java/de/adorsys/psd2/sandbox/xs2a/web/filter/MockCertificateFilter.java

```diff
@@ -21,22 +21,29 @@ public MockCertificateFilter(
   @Override
   public String getEncodedTppQwacCert(HttpServletRequest httpRequest) {
     return "-----BEGIN CERTIFICATE-----"
```

Comment:
it would be nice to add a possibility to read it from external file, so that next change can be done w/o recompilation (requires also documentation on this procedure)